templates/repo/issue/view_content/add_reaction.tmpl

```diff
@@ -1,13 +1,13 @@
 {{if .ctxData.IsSigned}}
 <div class="item action ui dropdown jump pointing top right select-reaction" data-action-url="{{.ActionURL}}">
-	<a class="add-reaction">
+	<a class="add-reaction muted">
```

Comment:
Why is there an `<a>` without a `href`?
If JS does something here, shouldn't it be a button instead so that it can even be focused and clicked accessibly?

Reply:
There is a lot of CSS depending on this being an `a` tag. The reactions are `a` as well, so I prefer to keep it to avoid a quite big refactor.

Reply:
Since the old code is so, I also do not want to be too strict about "tag" or "role" for this case.

At least, it doesn't harm anyone.